Khan/perseus
# Bugfix: first added locked figure settings accordion should be expanded on add

## Summary:
There was an issue where the first added locked figure settings would be collapsed, but all the settings are supposed to be expanded when added. This was happening because the initial expanded states array had one element in it when it was supposed to be empty. 

Issue: https://khanacademy.atlassian.net/browse/LEMS-1992

## Before

https://github.com/Khan/perseus/assets/13231763/4a1423d7-6f14-46dc-8431-f98ac8c64efe

## After

https://github.com/Khan/perseus/assets/13231763/2ab6a799-4216-4b29-9e3c-f3351b94dc32



## Test plan:
Storybook
- http://localhost:6006/?path=/story/perseuseditor-editorpage--demo
- Add an interactive graph widget
- Use a segment graph
- Click "Add locked element"
- Select any element
- The settings accordion should be expanded
